src/index.ts

```diff
@@ -1,6 +1,6 @@
 import SimpleSigner from './SimpleSigner'
 import NaclSigner from './NaclSigner'
-import { verifyJWT, createJWT, decodeJWT, Signer } from './JWT'
+import { verifyJWT, createJWT, decodeJWT, verifyJWS, createJWS Signer } from './JWT'
```

Comment:
This is missing a comma
```suggestion
import { verifyJWT, createJWT, decodeJWT, verifyJWS, createJWS, Signer } from './JWT'
```

Reply:
🤦